src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Monitor your Unreal Engine mobile app
+tags:
+    - Unreal Engine
+    - Mobile monitoring
+    - New Relic Mobile Unreal Engine
+    - Get started
+metaDescription: "New Relic's Unreal Engine for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+isTutorial: true
+freshnessValidatedDate: 2024-04-05
+---
+
+import mobileHybridSummary from 'images/mobile_screenshot-full_hybrid-summary.webp'
+
+import unrealGuidedInstall from 'images/newrelic_unreal_sdk_installation.webp'
+
+import unrealAgentAppTokenSetup from 'images/newrelic_unreal_sdk_configuration.webp'
+
+import mobileHandledExceptionsSummary from 'images/mobile_screenshot-full_handled-exceptions-summary.webp'
+
+Our New Relic Unreal Engine Plugin monitors your Unreal Engine mobile app and provides deep insights into your app's performance, errors, and user experience. Once you install and configure the Unreal Engine Plugin, you'll be able to:
+
+* **Capture C++ errors so you can quickly identify and fix problems.
+* **Create custom events and metrics:** Understand how your users interact with your app.
+
+<img
+  title="Unreal Engine summary view in the UI"
+  alt="Summary view of a Unreal Engine app in New Relic"
+  src={mobileHybridSummary}
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Summary**</DoNotTranslate>: View Unreal Engine data, and errors, and monitor how your app is performing over time.
+</figcaption>
+
+
+## (Recommended) Guided installation [#guided-install]
+
+To install the Unreal Engine Plugin, follow our [guided install](https://onenr.io/0gR70k5oZjo), located directly in the UI.
```

Comment:
@ndesai-newrelic This takes me to the Flutter setup, and when I tried to search for Unreal, nothing came up.